packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -597,10 +597,10 @@ export class BlobManager {
 			// If there is already an op for this storage ID, append the local ID to the list. Once any op for
 			// this storage ID is ack'd, all pending blobs for it can be resolved since the op will keep the
 			// blob alive in storage.
-			this.opsInFlight.set(response.id, [
-				...(this.opsInFlight.get(response.id) ?? []),
-				localId,
-			]);
+			this.opsInFlight.set(
```

Comment:
In other places we typically use a pattern like this, which I think is a little clearer to read.

```ts
let setForRemoteId = this.opsInFlight.get(response.id);
if (setForRemoteId === undefined) {
    setForRemoteId = new Set([localId]);
    this.opsInFlight.set(response.id, setForRemoteId);
}
```